src/components/DatePicker/DatePicker.tsx

```diff
@@ -17,6 +18,10 @@ export type Props = {
   isRangePicker?: boolean;
   /** A callback to return user selection */
   onChange?: (date: DateRange) => DateRange;
+  /** Option to disable some dates */
+  disableDates?: Modifier[];
+  /** Value to define if needed an initial state or to handle it externally */
+  value?: RangeModifier;
```

Comment:
Since we are adding a `value` prop that makes our component controlled from uncontrolled, shouldn't we also add an `onChange` function as prop that would give to our users the ability to update the value from where our datepicker is being used?
Not sure about my suggestion, because I don't know what's the case for this enhancement, but if we are ok for now with only the addition of `value` prop, I believe that we should consider adding an `onChange` function as an enhancement in the future, it would be really useful!

Reply:
onChange is there

Reply:
Missed it! 🙈